ideas/145-identity.md

```diff
@@ -0,0 +1,158 @@
+# Self-sovereign Identity
+
+## Preamble
+
+    Idea: #145
+    Title: Identity
+    Status: In Progress 
+    Created: 2018-05-04
+    Requires: #27 #73 ERC725 ERC735
+    
+## Summary
+
+Creates a common user-owned credentials for authentication on smart contracts and for enhancing users security with multisig and optional recovery.
+
+## Swarm Participants
```

Comment:
This sounds like a big project. Do you need other technical contributors?
What type of UX and PM support do you need?

Reply:
Project Manager should help integrate Identity in Status other projects, if any. I could also take the role of PM, but If anyone want to take place.
UX is also about integration of Identity, not sure what to comment about it

Reply:
I will externalize Recovery development in a new idea.